packages/template-retail-react-app/app/pages/checkout/partials/shipping-multi-address.jsx

```diff
@@ -0,0 +1,540 @@
+                                                                ...prev,
+                                                                [addressKey]: value
+                                                            }))
+                                                        }
+                                                    }}
+                                                    aria-labelledby={`delivery-address-label-${addressKey}`}
+                                                    size="md"
+                                                    borderColor="gray.300"
+                                                    _hover={{borderColor: 'gray.400'}}
+                                                    _focus={{
+                                                        borderColor: 'blue.500',
+                                                        boxShadow:
+                                                            '0 0 0 1px var(--chakra-colors-blue-500)'
+                                                    }}
+                                                >
+                                                    {addresses.map((addr) => (
+                                                        <option
+                                                            key={addr.addressId}
+                                                            value={addr.addressId}
+                                                        >
+                                                            {addr.firstName} {addr.lastName} -{' '}
+                                                            {addr.address1},{' '}
+                                                            {formatMessage(
+                                                                {
+                                                                    id: 'shipping_multi_address.format.address_line_2',
+                                                                    defaultMessage:
+                                                                        '{city}, {stateCode} {postalCode}'
+                                                                },
+                                                                {
+                                                                    city: addr.city,
+                                                                    stateCode: addr.stateCode || '',
+                                                                    postalCode: addr.postalCode
+                                                                }
+                                                            )}
+                                                        </option>
+                                                    ))}
+                                                    <option
+                                                        value="add-new"
+                                                        style={{
```

Comment:
Why do we need this but not other option? Becare of using hardcode value for styling here, it will be out of place for theme when customers are customising the theme object. `theme/foundations/color.js`